bedrock/products/models.py

```diff
@@ -103,6 +102,10 @@ class VPNResourceCenterDetailPage(ArticleDetailPageBase):
 
     template = "products/vpn/cms/resource-center/detail.html"
 
+    override_translatable_fields = ArticleDetailPageBase.override_translatable_fields + [
```

Comment:
**Question:** Do we also need to do this for the two snippets on the index page?

Reply:
Good catch!